src/lib/routing/index.test.ts

```diff
@@ -1,65 +1,107 @@
 import { describe, expect, test, vi } from 'vitest';
-import { defaultRedirectStatus, getRedirectTarget, redirectStatusCode } from '@lib/routing/redirects';
```

Comment:
I moved these to `redirects.test.ts` as these cover what's in `redirects.ts` and not what's in `index.ts`. Did that in a separate commit hoping git would understand this 🤷 .